spec/components/feedback/container_component_spec.rb

```diff
@@ -42,4 +43,23 @@
       expect(page).not_to have_link('Edit')
     end
   end
+
+  context 'when the feedback is not completed' do
+    it "renders 'Awaiting completion of feedback.'" do
+      render_inline(described_class.new(feedback: authored_feedback, current_user:))
+
+      expect(page).to have_content('Awaiting completion of feedback.')
+    end
+  end
+
+  context 'when the feedback is completed' do
+    let(:status) { :completed }
+
+    it 'renders the feedback rating out of 10' do
+      render_inline(described_class.new(feedback: authored_feedback, current_user:))
```

Comment:
this is really cool -- now I'm starting to see the beauty of view components. At first I was reading the spec and thought it was a system spec.